include/ace/managers/sprite.h

```diff
@@ -45,16 +49,20 @@ typedef struct tSprite {
  * @note This function doesn't handle the mouse input etc. automatically,
  * since one may want to control it with joy, keyboard or in other kind of way.
  *
- * @param pView View used for displaying sprites.
- * @param uwRawCopPos In raw mode, specifies an offset on where
+ * @param const tView *pView View used for displaying sprites.
```

Comment:
hmmm, all other doxycomments in ACE skip parameter types, and I'm not sure which way is better